File: src/main/java/org/example/bankramenserver/infrastructure/fcm/FcmProperties.java
package org.example.bankramenserver.infrastructure.fcm;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@Getter
@Setter
@Component
@ConfigurationProperties(prefix = "fcm")
public class FcmProperties {

    private boolean enabled = false;
    private String credentialsPath;
}

File: src/main/kotlin/org/example/balogserver/infrastructure/fcm/FcmProperties.kt
package org.example.balogserver.infrastructure.fcm

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

@Component
@ConfigurationProperties(prefix = "fcm")
class FcmProperties {
    var enabled = false
    var credentialsPath: String? = null
}
